Caddyfile

```diff
@@ -17,7 +17,43 @@
             browse
         }
     }
+
+    # Setup the regex for an beta match
+    @app_match {
+        path_regexp /beta/apps/hac-core(.*)
+    }
+    handle @app_match {
+        # Substitution for alias from nginx
+        uri strip_prefix /beta/apps/hac-core
+        file_server * {
+            root /opt/app-root/src/dist 
+            browse
+        }
+    }
+
+    # Setup the regex for an beta match
+    @app_match {
+        path_regexp /preview/apps/hac-core(.*)
+    }
+    handle @app_match {
+        # Substitution for alias from nginx
+        uri strip_prefix /preview/apps/hac-core
+        file_server * {
+            root /opt/app-root/src/dist 
+            browse
+        }
+    }
+
     handle / {
         redir /apps/chrome/index.html permanent
     }
+
+    handle /preview/ {
+        redir /preview/apps/chrome/index.html permanent
+    }
+
+    handle / {
```

Comment:
@karelhala we have this same handle twice (see just above) - is that expected? It routes to the same destination `redir /apps/chrome/index.html permanent`

Reply:
Ah, good point! I will change that.

Reply:
The last one was supposed to be `/beta` route.